react-select/react-select-tests.tsx

```diff
@@ -61,6 +63,30 @@ const CustomValue = React.createClass({
     }
 });
 
+const filterOptions = (options: Array<Option>, filter: string, values: Array<Option>) => {
```

Comment:
not sure why such an involved implementation is necessary for this test, but since you did it i'm going to review it 😄